GVFS/GVFS.Common/InstallerPreRunChecker.cs

```diff
@@ -69,36 +68,45 @@ public virtual bool TryUnmountAllGVFSRepos(out string consoleError)
                     return false;
                 }
 
-                // While checking for blocking processes like GVFS.Mount immediately after un-mounting,
-                // then sometimes GVFS.Mount shows up as running. But if the check is done after waiting
-                // for some time, then eventually GVFS.Mount goes away. The retry loop below is to help
-                // account for this delay between the time un-mount call returns and when GVFS.Mount
-                // actually quits.
-                this.tracer.RelatedInfo("Checking if GVFS or dependent processes are running.");
-                int retryCount = 10;
-                HashSet<string> processList = null;
-                while (retryCount > 0)
-                {
-                    if (!this.IsBlockingProcessRunning(out processList))
-                    {
-                        break;
-                    }
+                activity.RelatedInfo("Successfully unmounted repositories.");
+            }
 
-                    Thread.Sleep(TimeSpan.FromMilliseconds(250));
-                    retryCount--;
-                }
+            return true;
+        }
+
+        public virtual bool NoBlockingProcessCheck(out string consoleError)
+        {
+            consoleError = null;
 
-                if (processList.Count > 0)
+            this.tracer.RelatedInfo("Checking for running git processes.");
+
+            // While checking for blocking processes like GVFS.Mount immediately after un-mounting,
+            // then sometimes GVFS.Mount shows up as running. But if the check is done after waiting
+            // for some time, then eventually GVFS.Mount goes away. The retry loop below is to help
+            // account for this delay between the time un-mount call returns and when GVFS.Mount
+            // actually quits.
+            this.tracer.RelatedInfo("Checking if GVFS or dependent processes are running.");

```

Comment:
Nit:

Can one of these two be removed (maybe the first since we're not checking only for git)?

```
this.tracer.RelatedInfo("Checking for running git processes.");
this.tracer.RelatedInfo("Checking if GVFS or dependent processes are running.");
```